src/h5web/toolbar/controls/DomainSlider/hooks.ts

```diff
@@ -1,18 +1,18 @@
 import { useEffect, useState } from 'react';
+import { createMemo } from 'react-use';
 import type { Domain } from '../../../../packages/lib';
-import type { CustomDomain } from '../../../vis-packs/core/models';
-import { getSliderDomain } from './utils';
+import { getAppliedDomain, getSliderScale } from './utils';
 
-export function useAppliedDomain(
-  dataDomain: Domain,
-  customDomain: CustomDomain
-) {
-  const state = useState(getSliderDomain(dataDomain, customDomain));
-  const [, setAppliedDomain] = state;
+export const useAppliedDomain = createMemo(getAppliedDomain);
+export const useSliderScale = createMemo(getSliderScale);
+
+export function useSliderDomain(appliedDomain: Domain) {
+  const state = useState(appliedDomain);
+  const [, setDisplayedDomain] = state;
 
   useEffect(() => {
-    setAppliedDomain(getSliderDomain(dataDomain, customDomain));
-  }, [customDomain, dataDomain, setAppliedDomain]);
+    setDisplayedDomain(appliedDomain);
```

Comment:
`setDomain` perhaps ?

I feel that there are already enough `...Domain` in there...

Reply:
woops, sorry just a left over from a previous naming 😉

Reply:
`setState` even